### DISCUSSION TITLE
On Deck and Continue Watching

### DISCUSSION THREAD

**Discussion by BUGBEAR7** [croneter/PlexKodiConnect#1502] [General]
Helllo

In Plex player ( Web and App based) On Deck and Continue watching is combined. It shows the Next Episode to watch as well as On going TV Shows and Movies.

In Addon however On Deck (under Plex Hub) shows only TVshows . It shows the ongoing(to resume) TVShows as well as UP next Episodes of the TVShows which is currently being watched  it does not show the ongoing Movies.

I am not sure but in previous version it was same as Plex web or Media player. I am using v 3.2.0

**Comment by BUGBEAR7** [comment]
To explain this I am adding image for plex app and the PKC

Plex App from ios
https://i.imgur.com/7SlJptL.jpg
Plex Web
https://i.imgur.com/8Gcjff2.jpg


PKC On Deck
https://i.imgur.com/up4c0qf.png
PKC Continue Watching
https://i.imgur.com/DuHUPDo.png

**Comment by doomwithdon** [comment]
Any workaround to sync the continue watching section in Plex with a custom playlist in Kodi?
